Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1320,7 +1320,14 @@ def search_file_command(file):
     file_list = argToList(file)
     indicator_details = []
     for _file in file_list:
-        raw_res = search_indicator('sha256', _file)
+        try:
```

Comment:
why not to do lower to begin with?
And lets add a UT for this

Reply:
I looked it up and there are cases of "case sensitive" hash - https://stackoverflow.com/questions/23786939/what-kind-of-hash-returns-lower-and-upper-case-characters

This way they will also be supported as long as AutoFocus supports them.

Also I didn't find a way to test my change through a UT without changing more parts of the code - hence the addition of a command to the playbook as a replacement.

Reply:
seems this command can only get SHA256 which is not case sensitive - https://stackoverflow.com/questions/11892053/is-sha-256-case-insensitive